pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveInputFormatFragmenter.java

```diff
@@ -160,6 +166,89 @@ void verifySchema(Table tbl) throws Exception {
 
     }
 
+    private void compareTypes(DataType type, String hiveType, String fieldName) {
```

Comment:
This was moved to HiveUtilities.validateTypeCompatible when HAWQ-992 has been merged.